cmd/samToWig/testdata/test1.wig

```diff
@@ -0,0 +1,564 @@
+fixedStep chrom=chr1 start=1 step=1
+0
+0
+0
+0
+0
+0
+1.000000
```

Comment:
It goes beyond the scope of this pull request, but maybe we can change the print in wig to be %g so that we get "1" instead of "1.00000"